internal/cmdget/cmdget_test.go

```diff
@@ -379,9 +390,30 @@ func TestCmd_Execute_flagAndArgParsing(t *testing.T) {
 			r.Command.SilenceErrors = true
 			r.Command.SilenceUsage = true
 			r.Command.RunE = tc.runE
+			r.Command.PostRunE = nil
 			r.Command.SetArgs(tc.argsFunc(g.RepoDirectory, w.WorkspaceDirectory))
 			err := r.Command.Execute()
 			tc.validations(g.RepoDirectory, w.WorkspaceDirectory, r, err)
 		})
 	}
 }
+
+// addMergeCommentToExpected copies the expected path directory contents to
+// new temp directory and adds merge comment to the resources in directory
+// it also returns the cleanup function to clean the created temp directory
+func addMergeCommentToExpected(t *testing.T, path string) (string, func()) {
```

Comment:
Why are we adding this test here? it seems like it only tests the code in the `addmergecomment` package.

Reply:
Moved to testutil